events/validations/list.js

```diff
@@ -1,7 +1,57 @@
-const { query } = require('express-validator/check');
+const { query, checkSchema } = require('express-validator/check');
+const moment = require('moment');
 const ValidationHelper = require('../../util/ValidationHelper');
 
 module.exports = [
-  query('query[dojo_id]', 'dojo_id must be a UUID').isUUID(),
+  checkSchema({
+    'query[dojoId]': {
+      in: ['query'],
+      isUUID: true,
+    },
+    page: {
+      in: ['query'],
+      isInt: true,
+      toInt: true,
+      optional: true,
+    },
+    pageSize: {
+      in: ['query'],
+      isInt: true,
+      toInt: true,
+      optional: true,
+    },
+    'query[afterDate]': {
+      in: ['query'],
+      isISO8601: true,
+      customSanitizer: {
+        options(value, { req }) {
+          const utcOffset = req.query.utcOffset;
+          return moment.utc(value*1000).add(utcOffset, 'm').toISOString();
+        },
+      },
+      optional: true,
+    },
+    'query[beforeDate]': {
+      in: ['query'],
+      isISO8601: true,
+      customSanitizer: {
+        options(value, { req }) {
+          const utcOffset = req.query.utcOffset;
+          return moment.utc(value*1000).add(utcOffset, 'm').toISOString();
+        },
+      },
+      optional: true,
+    },
+    'utcOffset': {
```

Comment:
isn't that query[utcOffset]? If not, check the custom validator which make use of req.query.query